src/audio/igo_nr/igo_nr.c

```diff
@@ -790,6 +790,15 @@ static int32_t igo_nr_reset(struct comp_dev *dev)
 
 	cd->igo_nr_func = NULL;
 
+	cd->source_rate = 0;
+	cd->sink_rate = 0;
+	cd->source_frames = 0;
+	cd->sink_frames = 0;
+	cd->source_frames_max = 0;
+	cd->sink_frames_max = 0;
+	cd->frames = 0;
+	cd->invalid_params = false;
```

Comment:
invalid_param

Reply:
@eddy1021 you could make a fake header and stub out the functions so you can compile test these.

Reply:
Thanks! Fixed it and compiled locally.